### PR TITLE
add libraries/packages for android emulation and more tests

### PR DESCRIPTION
**Pull request by estute**
## Configuration Pull Request

Make sure that the following steps are done before merging
- [ ] @devops team member has commented with :+1:
- [ ] are you adding any new default values that need to be overriden when this goes live?  
  - [ ] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
  - [ ] Add an entry to the CHANGELOG.

### CODE REVIEW

**Comment by estute** [comment]
@jzoldak @jibsheet

**Comment by jzoldak** [comment]
LGTM 👍

File: playbooks/roles/jenkins_worker/tasks/android_testing_tools.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+
+# Install Python packages used for testing the edX Android app on Jenkins
+# Note: normally, we use a virtual environment for pip installs on a Jenkins 
+# worker, but since this is the only instance of python being used on the machine
+# we install them as system packages. If we start requiring more python, or
+# add other jobs to this worker, please reconsider this.
+- name: Install Android testing requirements using pip
```
Comment (jibsheet) [inline playbooks/roles/jenkins_worker/tasks/android_testing_tools.yml:8]: Is using a virtualenv significantly more complicated?  Normally we default to everything in a virtualenv in our jenkins, although I understand the specifics of this work.

**Review by jibsheet — APPROVED** [review]
One small question about "if virtualenv is easy, can we just do it?" but I understand if you want to avoid complexity for now.

**Pull request merged**